tests/adapters/switches/dell_test.py

```diff
@@ -462,6 +463,7 @@ def test_get_interfaces(self):
         assert_that(i2_x2.port_mode, is_(TRUNK))
         assert_that(i2_x2.trunk_native_vlan, is_(1500))
         assert_that(i2_x2.trunk_vlans, is_([900, 1000, 1001, 1003, 1004, 1005]))
+        assert_that(i2_x2.mtu, is_(5000))
```

Comment:
Please add

assert_that(....mtu, is_(None))

to ONE of the 3 other interfaces (i1_1) for example to test the case where the mtu is not in the config